web/impact/impact/settings.py

```diff
@@ -14,7 +14,7 @@
 
```

Comment:
I'm assuming that changes in this file are to be ignored. Please correct me if something (eek) is still importing from here.

Reply:
correct - we can safely ignore changes to this file.